Include/pyhash.h

```diff
@@ -12,8 +12,21 @@ PyAPI_FUNC(Py_hash_t) _Py_HashPointer(void*);
 PyAPI_FUNC(Py_hash_t) _Py_HashBytes(const void*, Py_ssize_t);
 #endif
 
-/* Prime multiplier used in string and various other hashes. */
-#define _PyHASH_MULTIPLIER 1000003UL  /* 0xf4243 */
+/* Constant used for various hashes */
+#define _PyHASH_MULTIPLIER ((Py_uhash_t)1000003UL)
```

Comment:
Why change this?  The tuple hash no longer uses it at all, so there's no need to change it.  I have no idea whether casting it to `Py_uhash_t` makes sense in places that still _do_ use it, and don't want to bother searching for them.  "Not broke, don't fix."

Reply:
Sure. I guess this was just a left-over from earlier changes.